samples/tasks.js

```diff
@@ -0,0 +1,135 @@
+  const cloudTasks = require('@google-cloud/tasks');
+
+  // Instantiates a client.
+  const client = new cloudTasks.CloudTasksClient();
+
+  // Construct the fully qualified queue name.
+  const parent = client.queuePath(project, location, queue);
+
+  // Prepare the payload.
+  const task = {
+    pullMessage: {
+      payload: Buffer.from('a message for the recipient').toString('base64')
+    }
+  };
+
+  // Construct the request body.
+  const request = {
+    parent: parent,
+    task: task
+  };
+
+  // Send create task request.
+  client.createTask(request).then(response => {
+    const task = response[0].name;
+    console.log(`Created task ${task}`);
+  }).catch(err => {
+    console.log(`Error in createTask: ${err.message || err}`);
+  });
+  // [END cloud_tasks_create_task]
+}
+
+function pullTask (project, location, queue) {
+  // [START cloud_tasks_pull_and_acknowledge_task]
+  // Imports the Google Cloud Tasks library.
+  const cloudTasks = require('@google-cloud/tasks');
+
+  // Instantiates a client.
+  const client = new cloudTasks.CloudTasksClient();
+
+  // Construct the fully qualified queue name.
+  const parent = client.queuePath(project, location, queue);
+
+  // Construct the request body.
+  const request = {
+    parent: parent,
+    leaseDuration: {
+      seconds: 600
+    },
+    maxTasks: 1,
+    responseView: 'FULL'
+  };
+
+  // Send lease task request.
+  client.leaseTasks(request).then(response => {
+    // Extract necessary properties to acknowledge task.
+    const {name, scheduleTime} = response[0].tasks[0];
+    console.log(`Leased task ${JSON.stringify({name, scheduleTime})}`);
+  }).catch(err => {
+    console.log(`Error in leaseTask: ${err.message || err}`);
+  });
+}
+
+function acknowledgeTask (task) {
+  // Imports the Google Cloud Tasks library.
+  const cloudTasks = require('@google-cloud/tasks');
+
+  // Instantiates a client.
+  const client = new cloudTasks.CloudTasksClient();
+
+  // Construct the request body.
+  const request = {
+    name: task.name,
+    scheduleTime: task.scheduleTime,
+    leaseDuration: {
+      seconds: 600
+    }
+  };
+
+  // Send acknowledge task request.
+  client.acknowledgeTask(request).then(() => {
+    console.log(`Acknowledged task ${task.name}`);
+  }).catch(err => {
+    console.log(`Error in acknowledgeTask: ${err.message || err}`);
+  });
+// [END cloud_tasks_pull_and_acknowledge_task]
+}
+
+require(`yargs`) // eslint-disable-line
+  .demand(1)
+  .command(
+    `create <project> <location> <queue>`,
+    `Create a task.`,
+    {},
+    (opts) => createTask(opts.project, opts.location, opts.queue)
+  )
+  .command(
+    `pull <project> <location> <queue>`,
+    `Pull a task.`,
+    {},
+    (opts) => pullTask(opts.project, opts.location, opts.queue)
+  )
+  .command(
+    `acknowledge <task>`,
+    `Acknowledge a task.`,
+    {},
+    (opts) => acknowledgeTask(JSON.parse(opts.task))
+  )
+  .example(`node $0 create my-project-id us-central1 my-queue`)
```

Comment:
Should these examples use curly braces? `${0}`

Reply:
I don't believe so. Other node samples just use $0.